internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -3163,6 +3213,36 @@ func flattenKubernetesClusterAPIAccessProfile(profile *managedclusters.ManagedCl
 	}
 }
 
+func expandBootstrapProfile(rawBootsrapProfile []interface{}) *managedclusters.ManagedClusterBootstrapProfile {
```

Comment:
Would you mind fixing this typo?

```suggestion
func expandBootstrapProfile(rawBootstrapProfile []interface{}) *managedclusters.ManagedClusterBootstrapProfile {
```